lib/sessions.js

```diff
@@ -9,6 +9,21 @@ module.exports = (app, config) => {
 
   const logger = config.logger || console;
 
+  app.use(cookieParser(config.session.secret, {
+    path: '/',
+    httpOnly: true,
+    secure: config.protocol === 'https'
+  }));
+
+  if (config.env === 'ci' && config.sessionStore) {
```

Comment:
it'd be nice if we can just inject a sessionStore for stuff that isn't just CI so maybe just remove that check. I get the intention I think it's a nice thing to have